src/services/offlineService.ts

```diff
@@ -50,9 +52,70 @@ export class OfflineService extends BaseService {
     this.log("Finished cleaning up offline files");
   }
 
-  public start() {
-    this.log("Run 'npm start' or 'func host start' to run service locally");
-    this.log("Make sure you have Azure Functions Core Tools installed");
-    this.log("If not installed run 'npm i azure-functions-core-tools -g")
+  /**
+   * Spawn `func host start` from core func tools
+   */
+  public async start() {
+    await this.spawn(configConstants.funcCoreTools, configConstants.funcCoreToolsArgs);
+  }
+
+  /**
+   * Spawn a Node child process with predefined environment variables
+   * @param command CLI Command - NO ARGS
+   * @param args Array of arguments for CLI command
+   * @param env Additional environment variables to be set in addition to
+   * predefined variables in `serverless.yml`
+   */
+  private spawn(command: string, args?: string[], env?: any): Promise<void> {
+    env = {
+      ...this.serverless.service.provider["environment"],
+      ...env
+    }
+    this.log(`Spawning process '${command} ${args.join(" ")}'`);
+    return new Promise((resolve, reject) => {
+      const childProcess = spawn(command, args, {env});
```

Comment:
I don't know if it has been made on purpose (if yes, that requires maybe some documentation), but passing the `env` variable here as an option is causing the default value `process.env` to not be used (and more particularly the PATH in it). Taking the PATH from `process.env` if it hasn't been set as an additional environment variable could be a solution.

https://nodejs.org/docs/latest-v10.x/api/child_process.html#child_process_child_process_spawn_command_args_options